src/components/molecules/AnnouncementBanner.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactElement } from 'react'
+import styles from './AnnouncementBanner.module.css'
+import Markdown from '../atoms/Markdown'
+
+export interface AnnouncementAction {
+  name: string
+  style?: 'text' | 'primary' | 'ghost'
+  handleAction: () => void
+}
+
+export default function AnnouncementBanner({
+  text,
+  action
+}: {
+  text?: string
+  action?: AnnouncementAction
+}): ReactElement {
+  return (
+    <div className={styles.container}>
+      <div className={styles.banner}>
+        {text && <Markdown className={styles.text} text={text} />}
+        {action && (
+          <button className={styles.button} onClick={action.handleAction}>
```

Comment:
we have `<Button style="text" size="small">...</Button>` so with that we should be able to do less style writing to overwrite default `button` style